### ISSUE TITLE
Ignoring an account still shows direct mentions in notifications

### ISSUE THREAD

**Issue by MrPetovan** [friendica/friendica#7644]
### Expected behavior

Ignoring a public account (no existing relationships) should suppress all notifications from them.

### Actual behavior

After ignoring a public account, at least direct mention notifications appear in the list.

### Steps to reproduce the problem

- Get into an argument online with a stranger on a contacts' thread.
- Ignore the stranger's account.
- Still getting mention notifications from them.

### Friendica version you encountered the problem

2019.09-rc

**Comment by AlfredSK** [comment]
"Steps to reproduce the problem:
Get into an argument online with a stranger
..."

😆

**Comment by MrPetovan** [comment]
It's a very important step, why would you ignore someone who would still want to mention you after you did?

**Comment by AlfredSK** [comment]
You're absolutely right. I'll try to reproduce it by following the steps.

**Comment by annando** [comment]
Ignoring is like: "Well, I don't really dislike you. If I would, I would block you. But I don't want to see your pots. (But you still can comment)"

AFAIK the corresponding definition table is saying it like this.

**Comment by MrPetovan** [comment]
Yes, and it should suppress notifications as well, as they are even more attention-demanding than posts/replies themselves.
